client/lib/analytics/sem.js

```diff
@@ -73,7 +73,12 @@ function setUtmCookie( name, value ) {
 	document.cookie = cookie.serialize( name, value, {
 		path: '/',
 		maxAge: UTM_COOKIE_MAX_AGE,
-		// domain: '.wordpress.com',
+		domain:
+			'.' +
+			document.location.hostname
+				.split( '.' )
+				.slice( -2 )
+				.join( '.' ),
```

Comment:
Brilliant, this does what we want, tested it with `calypso.localhost`, `en.support.wordpress.com`, `wordpress.com`, `es.wordpress.com` --> all return the last two parts of the domain 👍

Reply:
I don't think we need to support them at this time, but with domains like `wordpress.com.br` this wouldn't work as intended I think?

Reply:
Great point. It wouldn't work effectively in that case. Any domain with more than a single extension would include the second-level domain in the TLD itself, and therefore it would require custom logic to look at either (a) a known list of second-level TLDs, or (b) a specific name, such as "wordpress" existing in the third-level name.

Is it worth the overhead?

Reply:
> Is it worth the overhead?

Not at this point I think, but something to keep in mind -- maybe also worth adding a comment about in the code here.

Reply:
@lsinger Comment added. Thanks again!